packages/editor/src/components/post-publish-button/index.js

```diff
@@ -135,23 +134,12 @@ export class PostPublishButton extends Component {
 				( ! isPublishable && ! forceIsDirty ) ) &&
 			( ! hasNonPostEntityChanges || isSavingNonPostEntityChanges );
 
-		let publishStatus;
```

Comment:
I might have been missing something here, but it seems we only did that because we hadn't all the available `status` options in post editor.. I'll cc @jorgefilipecosta as I saw he has made changes in the past..